README.md

```diff
@@ -17,5 +17,6 @@ module "author" {
   aws_alb_listener_arn    = "${module.eq-ecs.aws_alb_listener_arn}"
   application_cidrs       = "${var.application_cidrs}"
   survey_launcher_url     = "${module.survey-launcher-for-ecs.survey_runner_launcher_address}"
+  schema_validator_url    = "${module.schema-validator-for-ecs.schema_validator_address}"
```

Comment:
you will need to PR `eq-terraform` with this change

Reply:
Yeah, I wasn't sure about this since it's just the README file. I noticed there's already a [schema-validator](https://github.com/ONSdigital/eq-terraform/blob/master/survey-runner.tf#L177) module in the eq-terraform repo. So does this mean I just need to update the README above to something like:
```
schema_validator_url    = "${module.schema-validator.schema_validator_address}"
```

or is there more to it than that?

Reply:
https://github.com/ONSdigital/eq-terraform/pull/115